website/versioned_docs/version-2.1.0/architecture/chaos-execution-plane.md

```diff
@@ -33,10 +33,18 @@ Litmus Backend Execution Infrastructure components orchestrate the execution of
 
 4. **Chaos Operator:** A Kubernetes custom-controller that manages the lifecycle of certain resources or applications intending to validate their "desired state". It helps reconcile the state of the ChaosEngine by performing specific actions upon CRUD of the ChaosEngine. It also defines a secondary resource (the ChaosEngine Runner pod), which is created & managed by it to implement the reconcile functions.
 
+<div style={{textAlign: 'center'}}>
+  <img src={require("../assets/chaos-execution-plane-chaos-operator.png").default} alt="Chaos Operator" />
+</div>
+
 5. **ChaosResult CR:** Holds the results of a chaos experiment, such as ChaosEngine reference, Experiment State, Verdict of the experiment (on completion), salient application/result attributes. It also acts as a source for metrics collection for observability.
 
 6. **Chaos Runner:** Acts as a bridge between the Chaos Operator and Chaos Experiments. It is a lifecycle manager for the chaos experiments that creates Experiment Jobs for the execution of experiment business logic and monitors the experiment pods (jobs) until completion.
 
+<div style={{textAlign: 'center'}}>
+  <img src={require("../assets/chaos-execution-plane-chaos-runner.png").default} alt="Chaos Runner" />
+</div>
+
```

Comment:
why the images are not visible here?

Reply:
@uditgaurav They are visible in my local server:
![Screenshot from 2022-01-14 20-15-14](https://user-images.githubusercontent.com/43271557/149534558-7b78a032-b752-4f51-9fe7-ef1f0fcbb052.png)
![Screenshot from 2022-01-14 20-15-21](https://user-images.githubusercontent.com/43271557/149534581-c0437d6c-5d19-495a-b6ae-2e6d72c8950d.png)